als-actions/shared/src/main/scala/org/mulesoft/als/actions/hover/PatchedHover.scala

```diff
@@ -1,34 +1,42 @@
 package org.mulesoft.als.actions.hover
 
 import amf.aml.client.scala.model.document.Dialect
+import amf.aml.client.scala.model.domain.ClassTerm
 import amf.core.client.common.position.{Range => AmfRange}
 import amf.core.client.scala.model.domain.AmfObject
 import amf.core.client.scala.vocabulary.ValueType
+import amf.core.internal.annotations.Aliases
 import org.mulesoft.als.common.YPartBranch
 import org.mulesoft.amfintegration.AmfImplicits.AmfObjectImp
 import org.mulesoft.amfintegration.dialect.dialects.asyncapi20.AsyncApi20Dialect
 import org.mulesoft.amfintegration.dialect.dialects.oas.{OAS20Dialect, OAS30Dialect}
 import org.mulesoft.amfintegration.dialect.dialects.raml.raml08.Raml08TypesDialect
 import org.mulesoft.amfintegration.dialect.dialects.raml.raml10.Raml10TypesDialect
-import org.mulesoft.amfintegration.vocabularies.integration.VocabularyProvider
+import org.mulesoft.amfintegration.vocabularies.integration.{AlsVocabularyRegistry, VocabularyProvider}
+import org.mulesoft.amfintegration.vocabularies.propertyterms.patched.raml.raml10.Raml10UsesKeyTerm
 import org.yaml.model.YMapEntry
 
 class PatchedHover(provider: VocabularyProvider) {
```

Comment:
make `private val provider:` as it is not going to be used past initialization

Reply:
I think in Scala if keyword `val` is not present in class fields, they are private by default.

So `private val provider: VocabularyProvider` is the same as `provider: VocabularyProvider`

Reply:
agus is right